mne/decoding/csp.py

```diff
@@ -652,3 +656,150 @@ def _ajd_pham(X, eps=1e-6, max_iter=15):
             break
     D = np.reshape(A, (n_times, -1, n_times)).transpose(1, 0, 2)
     return V, D
+
+
+class SPoC(CSP):
+    """Implementation of the SPoC spatial filtering with Covariance as input.
```

Comment:
I don't think you need to say "with covariance as input", it doesn't add much. It would be better to do:
```
"""
Source Power Comodulation (SPoC) spatial filtering.

SPoC [1]_ allows ...
```